tools/worker/worker_main.cc

```diff
@@ -17,10 +17,23 @@
 #include <string>
 #include <vector>
 
+#include "tools/cpp/runfiles/runfiles.h"
 #include "tools/worker/compile_with_worker.h"
 #include "tools/worker/compile_without_worker.h"
 
+using bazel::tools::cpp::runfiles::Runfiles;
+
 int main(int argc, char *argv[]) {
+  std::string error;
+  std::unique_ptr<Runfiles> runfiles(Runfiles::Create(argv[0], &error));
+  if (!runfiles) {
```

Comment:
The [sample code](https://github.com/bazelbuild/bazel/blob/bff973031a65c139a4878c87e5f335a895964ab9/tools/cpp/runfiles/runfiles_src.h#L44) uses this. Do we want to change it?
```suggestion
  if (runfiles == nullptr) {
```

Reply:
Was following the style of our codebase, but I don't know enough about C++, so I'll change it back.